packages/breadcrumb/src/react/index.js

```diff
@@ -1,38 +1,43 @@
 import * as glamor from 'glamor'
-import Button from '@pluralsight/ps-design-system-button/react'
-import Icon from '@pluralsight/ps-design-system-icon/react'
+import Button from '@pluralsight/ps-design-system-button/react.js'
+import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
+import Icon from '@pluralsight/ps-design-system-icon/react.js'
 import PropTypes from 'prop-types'
 import React from 'react'
 
-import css from '../css'
+import css from '../css/index.js'
 
 const styles = {
   breadcrumb: _ => glamor.css(css['.psds-breadcrumb'])
 }
 
 const Breadcrumb = React.forwardRef((props, ref) => {
-  const breadcrumbProps = {
-    ...styles.breadcrumb(props),
-    ...(props.style ? { style: props.style } : null),
-    ...(props.className ? { className: props.className } : null)
-  }
-
+  const { disabled, href, loading, onClick, ...rest } = props
   return (
-    <div {...breadcrumbProps}>
+    <div {...filterReactProps(rest)} {...styles.breadcrumb(props)}>
       <Button
-        {...props}
-        ref={ref}
-        icon={<Icon id={Icon.ids.caretLeft} />}
         appearance={Button.appearances.flat}
+        href={href}
```

Comment:
Pushing Button-bound props through to button explicitly.  Spreading everything else onto root.